_posts/2017-09-05-DevSwap.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: epic
+title: "DevSwap: An Experiment in Randomizing Teams"
+date: 2017-09-05
+categories: [culture, teams, people]
+author: [kana]
+---
+
+If you’ve read about Artsy's [Engineering Organization][org] before, you might know that we organize ourselves into product teams that serve different parts of our business. Today, Artsy employs 26 engineers, spread across four product teams—Publishing, Auctions, Collector GMV, and Partner Success. We also have a Platform team that is responsible for systems, infrastructure, and data, among other things. Across these teams, we share systems and apps. To stay cohesive with one another, we also have what we call “practices,” which are cross-team groups that meet and discuss shared tech.
+
+About a year and a half ago, we didn't have product teams. Back then, most engineers worked in a variety of stacks. As our business models changed, so did the way we organized ourselves. Interestingly, the number of engineers has not changed drastically, but the way we work has. We have greater focus now that we are specialized engineers, but that means there aren't as many opportunities to work in other stacks.
```

Comment:
Anyone know when we introduced product teams? I don't think 1.5 yrs ago is accurate.

Reply:
It's mid 2015 on the Artsy Timeline on the 24th floor